cross_list_courses/templates/list.html

```diff
@@ -120,7 +120,7 @@ <h4 class="modal-title bottom-spacing">Cross-Listed Courses in Current Terms:</h
             "bInfo" : false,
             "aoColumnDefs": [ { "orderable": false, "targets": 3 } ],
             "oLanguage": {
-              "sSearch": "Filter the list below"
+              "sSearch": "Filter the list"
```

Comment:
Same as above- was this change intended for cross listing tool?

Reply:
Ditto - I'll back these out